_data/administrations/federal.yml

```diff
@@ -3,158 +3,126 @@
   name:
     en: Agriculture and Agri-Food Canada
     fr: Agriculture et Agroalimentaire Canada
-
```

Comment:
Removed spaces because it was causing unnecessary diffs on commits when creating a new organization. It shouldn't be an issue for federal administration since we can't currently add a federal administration, but still changed it for consistency.